elements/AAC-850m6/update.js

```diff
@@ -611,6 +611,13 @@ var update = function(instance, properties, context) {
       }
     });
       
+    //if image is resized, updates the value right away instead of waiting for the doneTypingInterval timer  
+    $(`#${instance.data.id}`).mouseup(function(){
+      if ($($(`#${instance.data.id}`).children()[3]).children().length > 0){
```

Comment:
double jquery?

Reply:
by double do you mean that the same query object is referred to twice? Or do you mean the level of nested jquery?

Reply:
`$(`#${instance.data.id}:nth-child(3)`).children().length > 0` if that works